package.json

```diff
@@ -583,6 +583,26 @@
         "vim.cursorStylePerMode": {
           "type": "object",
           "description": "Customize cursor style per mode"
+        },
+        "vim.autoSwitchInputMethod.enable": {
+          "type": "boolean",
+          "description": "Switch input method automatically when mode changed",
+          "default": false
+        },
+        "vim.autoSwitchInputMethod.defaultIM": {
+          "type": "string",
+          "description": "The input method for your normal mode, find more information here:https://github.com/daipeihust/Vim#2find-your-default-input-method-key",
```

Comment:
Can you fix the link here and also have it point to VSCodeVim repo?

Reply:
Oh! I'm sorry. I forgot to change it to VSCodeVim repo.